lib/organisms/components/scalables/scalable1/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Fade from "react-reveal/Fade";
+import {
+  Box,
+  Text,
+  Heading,
+  Image,
+  Container,
+  Button,
+} from "../../../../atoms";
+import { FeatureBlock } from "../../../../molecules";
+import { SCALABLE_FEATURE } from "../../../../data/Crypto";
+import { ScalableWrapper, FeatureSection } from "./style";
+import ScalableImage from "../../../assets/image/crypto/scalable.jpg";
+import { GraphContent } from "@bcrumbs.net/bc-api";
+import withModelToDataObjProp from "../../../../../bootstrapers/showcase/utils/withModelToDataObjProp";
+
+interface ScalableHistoryProps {
+  row: object;
+  col: object;
+  title: object;
+  description: object;
+  btnStyle: object;
+  sectionSubTitle: object;
+  cardArea: object;
+  featureTitleStyle: object;
+  featureDescriptionStyle: object;
+  model: GraphContent;
+  isAR: boolean;
+  data: Record<string, string>;
+}
+const ScalableHistory = ({
+  row,
+  col,
+  title,
+  description,
+  btnStyle,
+  sectionSubTitle,
+  cardArea,
+  featureTitleStyle,
+  featureDescriptionStyle,
+  model,
+  isAR,
+  data,
+}: ScalableHistoryProps) => {
+  return (
+    <ScalableWrapper id="scalable" >
```

Comment:
```suggestion
    <ScalableWrapper id={model.name}>
    
```

I think we need to fix that for all sections of this template

Reply:
done